aws/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
```

Comment:
Did you intend to revert the prod/ dir in this PR?

Reply:
yes, it had no benefit now that we're using the same immage for our dev stack.

Reply:
ah sorry I just realized this wasn't done, somehow I cp instead of mv... my bad. Now done.